### PR TITLE
Add support for the Cosmos project

### PR DESCRIPTION
**Pull request by kant2002**
We have support for debugging, and want to make sure our integration with VS would be tighter then now. If you are not heard about us, you could read about Cosmos here: https://github.com/CosmosOS/Cosmos.

### CODE REVIEW

**Comment by yishaigalatzer** [comment]
After you sign the CLA, we can take this PR. Unfortunately we cannot take it into the NuGet 3 RTM, because we are locked down for contributions, this could make it to the next out of box release.

**Comment by kant2002** [comment]
Just sign the agreement

**Comment by yishaigalatzer** [comment]
@Eilon this seems trivial enough to take to RTM, can we take this?

**Comment by yishaigalatzer** [comment]
@kant2002 I was going to merge your changes today, it needs two tiny changes.
1. Add a trailing comma to the list (i'll mark in the PR)
2. Rebase your change against dev rather than master

File: src/PackageManagement.VisualStudio/Utility/EnvDTEProjectUtility.cs [hunk 1]
```diff
@@ -49,7 +49,8 @@ public static class EnvDTEProjectUtility
             NuGetVSConstants.NomadForVisualStudioProjectTypeGuid,
             NuGetVSConstants.TDSProjectTypeGuid,
             NuGetVSConstants.DxJsProjectTypeGuid,
-            NuGetVSConstants.DeploymentProjectTypeGuid
+            NuGetVSConstants.DeploymentProjectTypeGuid,
+            NuGetVSConstants.CosmosProjectTypeGuid
```
Comment (yishaigalatzer) [inline src/PackageManagement.VisualStudio/Utility/EnvDTEProjectUtility.cs]: Please add a trailing comma

File: src/PackageManagement.VisualStudio/Utility/EnvDTEProjectUtility.cs [hunk 2]
```diff
@@ -85,7 +86,8 @@ public static class EnvDTEProjectUtility
                 NuGetVSConstants.CppProjectTypeGuid,
                 NuGetVSConstants.SynergexProjectTypeGuid,
                 NuGetVSConstants.NomadForVisualStudioProjectTypeGuid,
-                NuGetVSConstants.DxJsProjectTypeGuid
+                NuGetVSConstants.DxJsProjectTypeGuid,
+                NuGetVSConstants.CosmosProjectTypeGuid
```
Comment (yishaigalatzer) [inline src/PackageManagement.VisualStudio/Utility/EnvDTEProjectUtility.cs]: trailing comma

**Comment by kant2002** [comment]
Sorry, I messed up with rebase. Create new pull request to the dev.

**Pull request closed without merging**